src/conduit/controllers/user.cljs

```diff
@@ -50,3 +50,12 @@
 
 (defmethod control :clear-errors [_ _ state]
   {:state (assoc state :errors nil)})
+
+(defmethod control :set-token [_ [token] state]
```

Comment:
it's not really obvious that action with name 'set-token' performs a query for fetching user info :)